src/bindings/config.ts

```diff
@@ -31,7 +33,7 @@ export const loadConfig = async (context: Context): Promise<BotConfig> => {
     payout: {
       chainId: process.env.CHAIN_ID ? Number(process.env.CHAIN_ID) : DEFAULT_CHAIN_ID,
       rpc: process.env.RPC_PROVIDER_URL || DEFAULT_RPC_ENDPOINT,
-      privateKey: process.env.UBIQUITY_BOT_EVM_PRIVATE_KEY || "",
+      privateKey: keyData ?? process.env.UBIQUITY_BOT_EVM_PRIVATE_KEY ?? "",
```

Comment:
If we make partners to encrypt their wallets' private keys then we don't need a separate private repo for a config because we can simply store an encrypted private key in a standard bot's config

Then this page https://github.com/ubiquity/pay.ubq.fi/pull/43 transforms into a single field which encrypts partner wallet's private key with the bot's public key

Reply:
The private repo is for additional secrecy, despite them being encrypted.
For the same reason, GitHub doesn't allow reading secret values through API (yes they're encrypted)

Reply:
Yes, if the bot's PK is leaked then all encrypted wallet PKs in public repos are at risk so it is better to have a private repo